### PR TITLE
Add Galaxy logo for operator CSV instead of Pulp logo

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

@denney-tech noticed that we still have the Pulp logo in the CSV rather than the Galaxy logo.  This PR adds a basic Galaxy logo.

image

This is a stop-gap for now, I need to reach out to some design folks and see if we can create a square logo that will look better.

### CODE REVIEW

**Comment by Denney-tech** [comment]
How about the official Galaxy icon?
[logos/galaxy/galaxy-icon.png at main · ansible/logos (github.com)](https://github.com/ansible/logos/blob/main/galaxy/galaxy-icon.png)
image

**Comment by rooftopcellist** [comment]
It would be nice to have something a little more specific to Galaxy, since the one you linked is generally associated with Ansible Core.  

We have another contender from the team:
* https://github.com/ansible/logos/blob/main/galaxy/galaxy-logo-ansibull.svg
image

**Comment by Denney-tech** [comment]
It's a png of the same favicon used for the official galaxy: https://galaxy.ansible.com/favicon.ico

**Comment by Denney-tech** [comment]
What about this?
galaxy-logo3

**Comment by rooftopcellist** [comment]
Our UI/UX folks like the idea of a simpler logo, but suggested we use this:

GalaxyIcon (1)

File: config/manifests/bases/galaxy-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -798,7 +798,8 @@ spec:
...
...LDEuMywwLjksMi4zbC0xLjUsMC41Yy0wLjItMC43LTAuNC0xLjMtMC43LTEuNmMtMC4zLTAuNC0wLjctMC43LTEuMi0xcy0xLjEtMC40LTEuOC0wLjQgICBjLTAuOCwwLTEuNCwwLjEtMiwwLjRzLTEsMC41LTEuNCwxUzEwLjIsMTEsMTAsMTEuNmMtMC4zLDAuOC0wLjUsMS43LTAuNSwyLjdjMCwxLjIsMC4yLDIuMywwLjYsMy4xYzAuNCwwLjgsMC45LDEuNSwxLjcsMS44ICAgczEuNSwwLjYsMi40LDAuNmMwLjgsMCwxLjQtMC4yLDIuMS0wLjVjMC43LTAuMywxLjItMC42LDEuNi0xdi0yLjVoLTMuOFYxNS42eiIKICAgICAgIGlkPSJwYXRoMTkiCiAgICAgICBzdHlsZT0ic3Ryb2tlOiMwMDAwMDA7c3Ryb2tlLXdpZHRoOjAuOTtzdHJva2UtbWl0ZXJsaW1pdDo0O3N0cm9rZS1kYXNoYXJyYXk6bm9uZSIgLz48cGF0aAogICAgICAgY2xhc3M9InN0MSIKICAgICAgIGQ9Ik0zNS44LDIxVjcuNGgxLjd2MTIuMWg2LjN2MS42TDM1LjgsMjFMMzUuOCwyMXoiCiAgICAgICBpZD0icGF0aDIxIgogICAgICAgc3R5bGU9InN0cm9rZTojMDAwMDAwO3N0cm9rZS13aWR0aDowLjk7c3Ryb2tlLW1pdGVybGltaXQ6NDtzdHJva2UtZGFzaGFycmF5Om5vbmUiIC8+PHBhdGgKICAgICAgIGNsYXNzPSJzdDEiCiAgICAgICBkPSJNMjEuNSwyMWw0LjktMTMuNmgxLjhMMzMuNSwyMWgtMkwzMCwxNi44aC01LjRMMjMuMiwyMUgyMS41eiBNMjQuNiwxNi44SDMwbC0xLjktNS4yICAgYy0wLjQtMS4yLTAuNy0yLjEtMC45LTIuOGMtMC4yLDAuOS0wLjQsMS43LTAuNywyLjZMMjQuNiwxNi44eiIKICAgICAgIGlkPSJwYXRoMjMiCiAgICAgICBzdHlsZT0ic3Ryb2tlOiMwMDAwMDA7c3Ryb2tlLXdpZHRoOjAuOTtzdHJva2UtbWl0ZXJsaW1pdDo0O3N0cm9rZS1kYXNoYXJyYXk6bm9uZSIgLz48L2c+PC9zdmc+Cg==
```
Comment (aknochow) [inline config/manifests/bases/galaxy-operator.clusterserviceversion.yaml]: Here's the code for the planet logo:
```
  - base64data: PHN2ZyB3aWR0aD0iMTI4IiBoZWlnaHQ9IjEyOCIgdmlld0JveD0iMCAwIDEyOCAxMjgiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxwYXRoIGQ9Ik02NC4wMjg1IDExMC41NTNDODkuOTA4NCAxMTAuNTUzIDExMC44ODggODkuNTM4IDExMC44ODggNjMuNjE0NkMxMTAuODg4IDM3LjY5MTIgODkuOTA4NCAxNi42NzYxIDY0LjAyODUgMTYuNjc2MUMzOC4xNDg3IDE2LjY3NjEgMTcuMTY4OSAzNy42OTEyIDE3LjE2ODkgNjMuNjE0NkMxNy4xNjg5IDg5LjUzOCAzOC4xNDg3IDExMC41NTMgNjQuMDI4NSAxMTAuNTUzWiIgZmlsbD0iIzc0QkFCRSIvPgo8cGF0aCBkPSJNNjQuMDMyNyAxNi42NzU2QzcyLjk3MyAxNi42NzU2IDgxLjkxMzMgMTkuMTQ2IDg5Ljc3NDYgMjQuMzk1N0MxMTEuMzU1IDM4Ljc1NTIgMTE3LjM2NiA2Ny43ODMgMTAzLjAzMSA4OS41NTM4QzkzLjkzNjUgMTAzLjI5NiA3OS4xMzg3IDExMC43MDcgNjMuODc4NSAxMTAuNzA3QzU0LjkzODIgMTEwLjcwNyA0NS45OTc5IDEwOC4yMzYgMzguMTM2NiAxMDIuOTg3QzE2LjU1NjUgODguNjI3MyAxMC41NDQ5IDU5LjU5OTYgMjQuODgwMyAzNy44Mjg4QzMzLjgyMDYgMjQuMDg2OSA0OC43NzI1IDE2LjY3NTYgNjQuMDMyNyAxNi42NzU2Wk02NC4wMzI3IDEzLjU4NzVDNDcuMjMxIDEzLjU4NzUgMzEuNTA4NCAyMS45MjUzIDIyLjI1OTggMzYuMTMwNEMxNC44NjEgNDcuMjQ3NCAxMi4zOTQ3IDYwLjY4MDQgMTUuMDE1MSA3My44MDQ2QzE3LjYzNTUgODYuOTI4OSAyNS4zNDI3IDk4LjIwMDMgMzYuNDQxIDEwNS42MTJDNDQuNjEwNiAxMTEuMDE2IDU0LjE2NzUgMTEzLjk0OSA2My44Nzg1IDExMy45NDlDODAuNjgwMiAxMTMuOTQ5IDk2LjQwMjggMTA1LjYxMiAxMDUuNjUxIDkxLjQwNjZDMTEzLjA1IDgwLjI4OTYgMTE1LjUxNyA2Ni44NTY1IDExMi44OTYgNTMuNzMyM0MxMTAuMjc2IDQwLjYwOCAxMDIuNTY5IDI5LjMzNjYgOTEuNDcwMiAyMS45MjUzQzgzLjMwMDYgMTYuNTIxMiA3My43NDM3IDEzLjU4NzUgNjQuMDMyNyAxMy41ODc1WiIgZmlsbD0iYmxhY2siLz4KPHBhdGggZD0iTTk3LjQ4MDEgMzAuNzI2Qzk1LjE2OCAyOC40MSA5Mi43MDE3IDI2LjI0ODMgODkuOTI3MSAyNC4zOTU1Qzg5LjE1NjQgMjMuOTMyMyA4OC4zODU2IDIzLjQ2OTEgODcuNjE0OSAyMy4wMDU5Qzg2LjY5MDEgMzMuOTY4NSA4MC42Nzg1IDQ5LjQwODggNzAuNTA1IDY0LjY5NDZDNTguNDgxOCA4My4wNjg2IDQzLjk5MjMgOTYuMDM4NCAzMy4zNTY0IDk5LjEyNjVDMzQuODk3OSAxMDAuMzYyIDM2LjQzOTMgMTAxLjU5NyAzOC4xMzQ5IDEwMi44MzJDNDAuMjkyOSAxMDQuMjIyIDQyLjQ1MDkgMTA1LjQ1NyA0NC43NjMgMTA2LjUzOEM1NS43MDcyIDk5LjU4OTcgNjguMTkyOSA4Ni43NzQyIDc4Ljk4MjkgNzAuNTYxOUM4OC4wNzc0IDU2LjUxMTMgOTQuMzk3MiA0Mi42MTUgOTcuNDgwMSAzMC43MjZaIiBmaWxsPSIjNEI5Njk2Ii8+CjxwYXRoIGQ9Ik00OS4yMzQ0IDEwOC4yMzZDNjguODEwNiAxMTQuNzIxIDkxLjE2MTQgMTA3LjYxOCAxMDMuMTg1IDg5LjU1MzNDMTE0LjQzNyA3Mi40MTQ2IDExMy4wNSA1MC42NDM4IDEwMS4zMzUgMzUuMjAzNUw0OS4yMzQ0IDEwOC4yMzZaIiBmaWxsPSIjNEI5Njk2Ii8+CjxwYXRoIGQ9Ik03My4yODU2IDE0LjM1OTVDNzQuMDU2MyAxNC41MTM5IDc0LjgyNyAxNC42NjgzIDc1LjU5NzcgMTQuODIyN0M4Mi44NDI1IDEwLjQ5OTQgODkuMTYyNCA5LjQxODYxIDkzLjQ3ODQgMTIuMTk3OUMxMDMuNjUyIDE4Ljk5MTYgOTcuNDg2MSA0Ni4xNjY1IDc5LjYwNTUgNzMuMTg3QzYxLjg3OSAxMDAuMjA4IDM5LjIxOTkgMTE2LjU3NCAyOS4wNDY0IDEwOS43OEMyNS4zNDcgMTA3LjMxIDIzLjgwNTYgMTAxLjkwNiAyNC4yNjggOTQuOTU3OEMyMy42NTE0IDk0LjE4NTggMjMuMTg5IDkzLjQxMzggMjIuNzI2NiA5Mi42NDE4QzE5LjE4MTMgMTA3LjE1NiAyMC40MTQ0IDExOC41ODEgMjcuMDQyNiAxMjMuMDU5QzM5LjgzNjUgMTMxLjU1MSA2Ny40MjgyIDExMS45NDIgODkuMDA4MiA3OS4zNjMxQzExMC40MzQgNDYuNzg0MSAxMTcuNTI1IDEzLjU4NzUgMTA0Ljg4NSA1LjI0OTczQzk3LjY0MDIgMC4zMDg4MzcgODUuOTI1MyA0LjMyMzMgNzMuMjg1NiAxNC4zNTk1WiIgZmlsbD0iI0I0REJEQSIvPgo8cGF0aCBkPSJNOTcuNjM5OSAzLjA4ODA2QzEwMC4yNiAzLjA4ODA2IDEwMi43MjcgMy43MDU2NyAxMDQuNzMxIDUuMDk1M0MxMTcuNTI0IDEzLjU4NzUgMTEwLjQzNCA0Ni43ODQxIDg4Ljg1MzcgNzkuMjA4N0M3MC44MTkgMTA2LjY5MiA0OC4zMTQgMTI0Ljc1OCAzMy45Nzg3IDEyNC43NThDMzEuMzU4MyAxMjQuNzU4IDI4Ljg5MiAxMjQuMTQgMjYuODg4MSAxMjIuNzVDMjAuMTA1OCAxMTguMjczIDE5LjAyNjggMTA3LjAwMSAyMi41NzIxIDkyLjMzMjlDMjMuMDM0NSA5My4xMDQ5IDIzLjY1MTEgOTMuODc3IDI0LjExMzUgOTQuNjQ5QzIzLjY1MTEgMTAxLjc1MSAyNS4xOTI1IDEwNy4wMDEgMjguODkyIDEwOS40NzJDMzAuNDMzNCAxMTAuNTUyIDMyLjI4MzEgMTExLjAxNiAzNC4yODcgMTExLjAxNkM0NS44NDc3IDExMS4wMTYgNjQuMzQ0OSA5NS43Mjk4IDc5LjQ1MSA3Mi44NzgyQzk3LjE3NzUgNDUuODU3NiAxMDMuMzQzIDE4LjY4MjcgOTMuMzIzOSAxMS44ODlDOTEuNzgyNSAxMC44MDgyIDg5LjkzMjcgMTAuMzQ1IDg3LjkyODkgMTAuMzQ1Qzg0LjM4MzYgMTAuMzQ1IDgwLjA2NzYgMTEuODg5IDc1LjQ0MzMgMTQuNTEzOUM3NC42NzI2IDE0LjM1OTUgNzMuOTAxOCAxNC4yMDUxIDczLjEzMTEgMTQuMDUwN0M4Mi4zNzk3IDcuMTAyNTQgOTAuODU3NiAzLjA4ODA2IDk3LjYzOTkgMy4wODgwNlpNOTcuNjM5OSAwQzkwLjI0MSAwIDgxLjE0NjYgNC4xNjg4NyA3MS4yODE0IDExLjg4OUw2NS43MzIyIDE2LjIxMjNMNzIuNjY4NyAxNy4yOTMxQzczLjI4NTMgMTcuNDQ3NSA3NC4wNTYgMTcuNjAxOSA3NC44MjY3IDE3Ljc1NjNMNzYuMDU5OCAxOC4wNjUxTDc3LjEzODggMTcuNDQ3NUM4MS4zMDA3IDE0Ljk3NzEgODUuMTU0MyAxMy43NDE5IDg3LjkyODkgMTMuNzQxOUM4OS40NzAzIDEzLjc0MTkgOTAuNzAzNSAxNC4wNTA3IDkxLjYyODMgMTQuNjY4M0M5OS4xODE0IDE5LjYwOTIgOTQuODY1MyA0NC4xNTkyIDc2Ljk4NDcgNzEuMzM0MUM2MS4xMDc5IDk1LjQyMSA0My41MzU2IDEwOC4wODIgMzQuNDQxMSAxMDguMDgyQzMyLjg5OTcgMTA4LjA4MiAzMS42NjY1IDEwNy43NzMgMzAuNzQxNyAxMDcuMTU2QzI4LjEyMTMgMTA1LjQ1NyAyNi44ODgxIDEwMS4xMzQgMjcuMzUwNSA5NS4xMTIyVjk0LjAzMTRMMjYuNzM0IDkzLjEwNDlDMjYuMjcxNSA5Mi40ODczIDI1LjY1NDkgOTEuNzE1MyAyNS4xOTI1IDkwLjk0MzNMMjEuMzM4OSA4NC45MjE2TDE5LjY0MzQgOTEuODY5N0MxNS42MzU2IDEwOC41NDUgMTcuNjM5NSAxMjAuNDM0IDI1LjM0NjcgMTI1LjUzQzI3LjgxMyAxMjcuMjI4IDMwLjc0MTcgMTI4IDM0LjEzMjggMTI4QzUwLjAwOTYgMTI4IDcyLjk3NyAxMDkuMTYzIDkxLjYyODMgODEuMDYxNUMxMDIuMjY0IDY1LjAwMzYgMTA5LjY2MyA0OC4zMjgxIDExMi43NDYgMzMuODE0MkMxMTUuOTgzIDE4LjUyODQgMTEzLjgyNSA3LjQxMTMzIDEwNi41OCAyLjYyNDg0QzEwMy44MDYgMC43NzIwMSAxMDAuODc3IDAgOTcuNjM5OSAwWiIgZmlsbD0iYmxhY2siLz4KPC9zdmc+Cg==
  ```
  
Which looks like this:

image

**Comment by rooftopcellist** [comment]
I just updated the PR with the https://github.com/ansible/logos/blob/main/galaxy/galaxy-icon.svg logo and checked that it works in the CSV.

**Review by aknochow — APPROVED** [review]

**Pull request merged**